RealmSwift/Tests/ListTests.swift

```diff
@@ -299,20 +299,20 @@ class ListTests: TestCase {
 
         array.append(objectsIn: [str1, str2])
 
-        array.swap(index1: 0, 1)
+        array.swap(0, 1)
         XCTAssertEqual(Int(2), array.count)
         assertEqual(str2, array[0])
         assertEqual(str1, array[1])
 
-        array.swap(index1: 1, 1)
+        array.swap(1, 1)
         XCTAssertEqual(Int(2), array.count)
         assertEqual(str2, array[0])
         assertEqual(str1, array[1])
 
-        assertThrows(array.swap(index1: -1, 0))
-        assertThrows(array.swap(index1: 0, -1))
-        assertThrows(array.swap(index1: 1000, 0))
-        assertThrows(array.swap(index1: 0, 1000))
+        assertThrows(array.swap(-1, 0))
```

Comment:
In Swift 4, the equivalent method on `Array` appears to be called [`swapAt`](https://developer.apple.com/documentation/swift/array/2893281-swapat).